### ISSUE TITLE
FEATURE: similar items

### ISSUE THREAD

**Issue by lenzai** [coderbunker/inventory-server#17]
bugs and UX flaws , require in-depth redesign of this feature:

- displaying broken link to items with no ID.
- except for a few specific items, such as printers, picture are not helpful to identify the similar item.
- we need to consider item model and location as key discrimination factor for the user.

Let's talk here.

**Comment by mattermoran** [comment]
What do you mean by items with no ID? If ID is not in the spreadsheet?

**Comment by mattermoran** [comment]
For images, we could use google images to find the right image for each item

**Comment by Spazcool** [comment]
@mattermoran that's correct, over half of the items in the spreadsheet do not have UUID. So the link defaults to the root url (ie your react page with the camera on it)

**Comment by mattermoran** [comment]
Well then solution I have here should work just fine: https://github.com/coderbunker/inventory-server/issues/14

**Comment by mattermoran** [comment]
Because it would compare the link to the ones we already store and then decide to show the item page or display broken link

**Comment by mattermoran** [comment]
Doug right, this issue, https://github.com/coderbunker/inventory-server/issues/16 and https://github.com/coderbunker/inventory-server/issues/14 basically the same

**Comment by lenzai** [comment]
@mattermoran 
- images
finding missing images is one step (cf. project board cards, feel free to add comments there or convert it to an issue.)
But the real challenge here is that even with images, the Similar items feature would not really be usable. 

- broken link
#16 is the issue. 
#14 is a confusing discussion. I am still not sure what "broken ID" means besides missing ID.

@Spazcool  let's make the redirect to camera page a separate issue.
